web/packages/teleterm/src/ui/ClusterConnect/ClusterLogin/FormLogin/PromptWebauthn/PromptWebauthn.tsx

```diff
@@ -98,7 +98,7 @@ function PromptCredential({
                 &:hover,
                 &:focus {
                   cursor: pointer;
-                  background: ${props => props.theme.colors.action.hover};
```

Comment:
With this change, I think we don't user `action.*` colors anywhere in the app. I think we could just remove them? cc @rudream